implementations.md

```diff
@@ -0,0 +1,13 @@
+# Implementations of Container Specification
+
+## Runtime
+
+* [opencontainers/runc](https://github.com/opencontainers/runc) - reference implementation of OCF runtime
+* [hyperhq/runv](https://github.com/hyperhq/runv) - Hypervisor-based runtime for OCF
+* [huawei-openlab/ocp-testing](https://github.com/huawei-openlab/ocp-testing) - Testing framework for OCF runtime
+
+## Configuration
```

Comment:
We con drop it for now. In my mind they were different aspects, but I realize their basically overlapped for now.